BIDMessaging.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2018, 1Kosmos Inc. All rights reserved.
+ * Licensed under 1Kosmos Open Source Public License version 1.0 (the "License");
+ * You may not use this file except in compliance with the License. 
+ * You may obtain a copy of this license at 
+ *    https://github.com/1Kosmos/1Kosmos_License/blob/main/LICENSE.txt
+ */
+
+"use strict";
+const { v4: uuidv4 } = require('uuid');
+
+const BIDECDSA = require('./BIDECDSA');
+const BIDSDK = require('./BIDSDK');
+const fetch = require('node-fetch');
+
+const sendSMS = async (smsTo, smsISDCode, smsTemplate, sessionUrl) => {
+    try {
```

Comment:
sendSMS function has nothing to do with "sessinoUrl"
The caller is responsible for sending a pre-formatted text message which should be sent to downstream service as as a b64 string.

Reply:
okay got it